extensions/cornerstone/src/initDoubleClick.ts

```diff
@@ -0,0 +1,67 @@
+import { eventTarget, EVENTS } from '@cornerstonejs/core';
+import { Enums } from '@cornerstonejs/tools';
+import { CommandsManager, CustomizationService, Types } from '@ohif/core';
+import { findNearbyToolData } from './utils/findNearbyToolData';
+
+const cs3DToolsEvents = Enums.Events;
+
+const DEFAULT_DOUBLE_CLICK_COMMAND: Types.Command = {
+  commandName: 'toggleOneUp',
+  commandOptions: {},
+};
+
+export type initDoubleClickArgs = {
+  customizationService: CustomizationService;
+  commandsManager: CommandsManager;
+};
+
+function initDoubleClick({
+  customizationService,
+  commandsManager,
+}: initDoubleClickArgs): void {
+  const cornerstoneViewportHandleDoubleClick = (evt: CustomEvent) => {
+    // Do not allow double click on a tool.
+    const nearbyToolData = findNearbyToolData(commandsManager, evt);
+    if (nearbyToolData) {
+      return;
+    }
+
+    // Allows for the customization of the double click on a viewport.
+    const customizations: Types.Command | Types.CommandCustomization =
```

Comment:
Could you use:
customizationService.get('cornerstoneViewportClickCommands')
and get the key modifiers, so that one can bind shiftdoubleClick etc, all as part of the single configuration?
I do REALLY appreciate you using the customization service to do, as that sort of thing can be highly specific to viewer configurations.